recipes/cromwell/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "40" %}
+{% set sha256_cromwell = "8e1f9c8777bb3b860fd1bf6b6dc05a51145889620f0510cb2fa89aa7e1dfd3ec" %}
+{% set sha256_womtool = "1904ebc87c605b4330ddd88770b522104d0bd506be2b9bcfe8424e90d38e4fb9" %}
+
+package:
+  name: cromwell
+  version: 0.{{ version }}
```

Comment:
Is there a reason for prepending `0` here?

Reply:
History reasons. This package is moved from Bioconda. The package has been in bioconda for quite some time. I don't know about the motivation of the original writer of the package. 
The move to conda-forge is a nice opportunity to bring the version in line with the actual version number.